plugins/ledmatrix-weather/manager.py

```
@@ -691,14 +737,18 @@ def parse_ts(s):
             moon_phase = astral_moon.phase(d) / 28.0
             try:
                 moonrise_dt = astral_moon.moonrise(observer, d, tzinfo=tz)
-                moonrise_ts = int(moonrise_dt.timestamp()) if moonrise_dt else None
             except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)